### ISSUE TITLE
lables is not set for DNS service.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#2302]
Rancher server version - v0.41.0-rc1
Rancher compose version - rancher-compose-v0.4.1-rc2

Used the following docker-compose.yml to create a DNS service with label.

```
WEB1-label:
  labels:
    foo: bar
  image: rancher/dns-service
```

DNS service created does not have labels set:

```
"kind": "dnsService",
"launchConfig": {
"networkMode": "managed",
"privileged": false,
"publishAllPorts": false,
"readOnly": false,
"startOnCreate": true,
"stdinOpen": false,
"tty": false,
"version": "0",
},
```

**Comment by deniseschannon** [comment]
With the re-factor in v0.5.0 of rancher-compose for in-service upgrade (how we handle starting services), this should be fixed already in v0.5.0.

**Comment by sangeethah** [comment]
Tested with rancher-compose-v0.5.0 and rancher-server version v0.43.1

Lables are now set for DNS service when creating DNS service with label from rancher-compose.
